suite-native/module-accounts-import/src/components/QRWithLaser.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+
+import {
+    Canvas,
+    Group,
+    ImageSVG,
+    LinearGradient,
+    Mask,
+    Rect,
+    RoundedRect,
+    RoundedRectProps,
+    useComputedValue,
+    useLoop,
+    useSVG,
+    vec,
+} from '@shopify/react-native-skia';
+
+import { icons } from '@trezor/icons';
+import { useNativeStyles } from '@trezor/styles';
+
+const ROUNDED_CORNER_SIZE = 15;
+
+type RoundedCornerProps = {
+    x?: number;
+    y?: number;
+    width: number;
+    height: number;
+} & Omit<RoundedRectProps, 'x' | 'y' | 'width' | 'height'>;
+
+const RoundedCorner = ({ x = 0, y = 0, width, height, ...restProps }: RoundedCornerProps) => {
+    const cornerSize = 24;
+    const strokeWidth = 1;
```

Comment:
nit: this could be outside the component and using snake case

Reply:
I want to keep this values here because they may be dynamic in future (because of responsibility or something)